packages/ckeditor5-core/src/editor/editor.js

```diff
@@ -182,7 +182,8 @@ export default class Editor {
 		 * @readonly
 		 * @member {module:engine/controller/editingcontroller~EditingController}
 		 */
-		this.editing = new EditingController( this.model, stylesProcessor );
+		const sourceElementRoot = config.sourceElementRoot ? config.sourceElementRoot : document; // eslint-disable-line no-undef
```

Comment:
long line. can you put the comment on the line before with // eslint-disable-next-line?